components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OIDCAuthenticatorConstants.java

```diff
@@ -44,7 +44,7 @@ private OIDCAuthenticatorConstants() {
     public static final String OAUTH2_AUTHZ_URL = "OAuth2AuthzEPUrl";
     public static final String OAUTH2_TOKEN_URL = "OAuth2TokenEPUrl";
     public static final String IS_BASIC_AUTH_ENABLED = "IsBasicAuthEnabled";
-
+    public static final String GOOGLE_ONE_TAP_ENABLED = "IsGoogleOneTapEnabled";
```

Comment:
What is the use of this contant? Its better if we exclude google constants from this class since this class does not say anything about google.

Reply:
Fixed 7f1d7ec6afa72c7edeb5489211498b3660bda6be